ingestion/src/metadata/utils/secrets/manage_ssl.py

```diff
@@ -0,0 +1,166 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+"""
+Module to manage SSL certificates
+"""
+import os
+import tempfile
+from functools import singledispatch, singledispatchmethod
+from typing import Optional, Union, cast
+
+from metadata.generated.schema.entity.services.connections.dashboard.qlikSenseConnection import (
+    QlikSenseConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.dorisConnection import (
+    DorisConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.greenplumConnection import (
+    GreenplumConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.mysqlConnection import (
+    MysqlConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.postgresConnection import (
+    PostgresConnection,
+)
+from metadata.generated.schema.entity.services.connections.database.redshiftConnection import (
+    RedshiftConnection,
+)
+from metadata.generated.schema.entity.services.connections.messaging.kafkaConnection import (
+    KafkaConnection,
+)
+from metadata.generated.schema.security.ssl import verifySSLConfig
+from metadata.ingestion.connections.builders import init_empty_connection_arguments
+
+
+class SSLManager:
```

Comment:
nit - the file name could be `ssl_manager.py`. Also not sure i'd place it under `secrets` since there we only have secrets manager stuff. Maybe directly under `utils` or `source`?